code/modules/antagonists/hypnotization/hypnotized.dm

```diff
@@ -0,0 +1,100 @@
+/proc/hypnotize(mob/living/carbon/victim, hypnotic_phrase)
+	if(!victim.mind)
+		return
+	message_admins("[ADMIN_LOOKUPFLW(victim)] was hypnotized with the phrase '[hypnotic_phrase]'.")
+	victim.log_message("was hypnotized with the phrase '[hypnotic_phrase]'.", LOG_ATTACK, color="red")
+	log_game("[key_name(victim)] was hypnotized with the phrase '[hypnotic_phrase]'.")
+	to_chat(victim, "<span class='reallybig hypnophrase'>[hypnotic_phrase]</span>")
+	to_chat(victim, "<span class='notice'>[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
+												"Your head hurts, but this is all you can think of. It must be vitally important.",\
+												"You feel a part of your mind repeating this over and over. You need to follow these words.",\
+												"Something about this sounds... right, for some reason. You feel like you should follow these words.",\
+												"These words keep echoing in your mind. You find yourself completely fascinated by them.")]</span>")
+	to_chat(victim, "<span class='boldwarning'>You've been hypnotized by this sentence. You must follow these words. If it isn't a clear order, you can freely interpret how to do so,\
+										as long as you act like the words are your highest priority.</span>")
+	var/atom/movable/screen/alert/hypnosis/hypno_alert = victim.throw_alert("hypnosis", /atom/movable/screen/alert/hypnosis)
+	hypno_alert.desc = "\"[hypnotic_phrase]\"... your mind seems to be fixated on this concept."
+	var/datum/mind/M = victim.mind
+	var/datum/antagonist/hypnotized/B = new()
+	var/datum/objective/hypnotized/objective = new(hypnotic_phrase)
+	B.objectives += objective
+	log_objective(M, objective.explanation_text)
+	M.add_antag_datum(B)
+	var/begin_message = "<span class='deadsay'><b>[victim]</b> has been hypnotized with the following phrase: "
+	var/obj_message = hypnotic_phrase
+	var/end_message = "</b>.</span>"
```

Comment:
this is really ugly, just replace it with `[]` template strings. i know it's copypasta or whatever but might as well